src/Items/ItemFishingRod.h

```diff
@@ -143,55 +148,57 @@ class cItemFishingRodHandler :
 				}
 				else if (ItemCategory <= 14)  // Junk 10%
 				{
-					int Junk = Random.RandInt(70);
-					if (Junk <= 1)
+					int Junk = Random.RandInt(82);
+					if (Junk < 10)  // 10 / 83 chance of spawning a bowl
```

Comment:
This seems ideal for a [`std::discrete_distribution`](http://en.cppreference.com/w/cpp/numeric/random/discrete_distribution).